src/webgpu/util/floating_point.ts

```diff
@@ -4908,7 +4928,7 @@ class F16Traits extends FPTraits {
   public readonly negationInterval = this.unimplementedScalarToInterval.bind(this);
   public readonly normalizeInterval = this.unimplementedVectorToVector.bind(this);
   public readonly powInterval = this.unimplementedScalarPairToInterval.bind(this);
-  public readonly quantizeToF16Interval = this.unimplementedScalarToInterval.bind(this);
```

Comment:
Sorry for missing that this wasn't defined for all of the fp types.

I would prefer to remove quanitizeToF16Interval from FPTraits entirely, and instead only implement it in the FPTraits subclasses that actually define it (F32Traits and eventually AbstractFloatTraits), so that trying to use it on FPTraits will be a compile time error instead of a runtime error.

There is already various unpack*Interval functions that do this in F32Traits, as an example.

Reply:
Done. I change run*Op in `FPTraits` from `private` to `protected`.